hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/regression/TargetNetworkPrep.java

```diff
@@ -174,29 +170,6 @@ private HapiSpec ensureSystemStateAsExpected() {
         }
     }
 
-    private HapiSpec ensureBlockedAccountIsCreated() {
```

Comment:
Q: are we losing coverage here?
If so can it not be controlled by the property?

Reply:
We have coverage on unit test level, here we were verifying that when we have the flag enabled the node will be blocking accounts from the list.
Unfortunately for this case overriding the property in the e2e test will not work because the blocklisting logic is run on node startup when the node handles the first arriving transaction, so even if we try to override it here it will not matter.